references/V1_OutboundCallback.yml

```diff
@@ -334,6 +338,84 @@ definitions:
       - messageId
       - annotationId
       - time
+  AddReactionBody:
+    type: object
+    description: |
+      Notifies the addition of an emoji reaction to a message.
+
+      This event is only sent to webhooks that
+      - have been added for the reaction-added event
+      - and belong to an app that is a member of the space.
+    properties:
+      type:
+        description: The event type is `reaction-added`.
+        type: string
+      spaceId:
+        description: Id of the space in which the reaction was added.
+        type: string
+      objectId:
+        description: Id of the object being reacted to. Currently, only messages can be reacted to.
+        type: string
+      objectType:
+        description: The type of object in the space being reacted to. Currently, the only supported objectType is MESSAGE.
+        type: string
+      time:
+        description: "Time and date of the reaction being added, in milliseconds since January 1st, 00:00, 1970 UTC"
+        type: integer
+        format: int64
+      reaction:
+        description: "The reaction that was added."
+        type: string
+      userId:
+        description: "Id of the user that added the reaction."
+        type: string

```

Comment:
Some descriptions have `"` and some don't - we may want to keep consistent (haven't looked at what's the common behavior in others).